relnotes/version.feature.md

```diff
@@ -0,0 +1,21 @@
+### Print the application's version string to the control socket
+
+`ocean.util.app.DaemonApp`
+
+`DaemonApp`'s will now print the application's version string to the control
+unix socket when `show_version` command is received:
+
+```
+$ echo 'show_version' | nc -U /srv/dlsnode/dlsnode.socket dlsnode
+version v1.9.0-28-gcbc6-dirty (compiled by 'Nemanja Boric' on 2018-03-27
```

Comment:
Note that this will change after merging into v4 branch as we changed `getVersionString` to only print basic version information and introduced `getBuildInfoString` for a more detailed one.

How shall it behave in v4?

Reply:
It uses `getVersionString`, so it will print the basic version information. Does the `--version` behave in the same way (does it do different things in v4 and v3)? If so, I think we're fine. If not, I'll avoid breaking change.

Reply:
Yes, it was intentional breaking change for `--version` in v4 - and I am asking if this feature should follow or do something different.